### PR TITLE
[DEV-5674] DEFC Filter Styling Fixes

### PR DESCRIPTION
**Pull request by parikh-adit**
**High level description:**

Fix missing labels in DEFC checkbox. 

**Technical details:**

Added prop "subLabel" and styling to change both labels. 

**JIRA Ticket:**
[DEV-5674 ](https://federal-spending-transparency.atlassian.net/browse/DEV-5674)

**Mockup:**
https://federal-spending-transparency.atlassian.net/browse/DEV-5674

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
`N/A` Verified mobile/tablet/desktop/monitor responsiveness
`N/A` Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
`N/A` Added Unit Tests for methods in Container Components, reducers, helper functions, and models `if applicable`

Reviewer(s):
`N/A` Design review complete `if applicable`
- [x] Code review complete

### CODE REVIEW

**Comment by MaxwellKendall** [comment]
@parikh-adit there's also an issue with the filter on IE. This should be part of the A/C of that ticket:

https://files.slack.com/files-pri/TE1MMJVH6-F016ZKUGZ63/screen_shot_2020-07-20_at_4.54.24_pm.png

File: src/_scss/pages/search/filters/otherFilters/_defCheckbox.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 .def-code-filter {
+    margin-bottom: rem(12);
+    padding: rem(12);
     .def-checkbox-tree {
+      margin-bottom: 2rem;
+      border-bottom: rem(1) solid rgb(214, 215, 217);
```
Comment (ebdabbs) [inline src/_scss/pages/search/filters/otherFilters/_defCheckbox.scss:6]: Not sure if design has looked at this, but there is a lot of padding on the left side of the checkbox tree (more so than before and other checkbox tree sections). If we eliminated some of that (right in screenshot below), the labels could fit without scrolling and the second border-bottom may not be necessary.
defcPadding
checkboxAlignment

Comment (MaxwellKendall) [reply src/_scss/pages/search/filters/otherFilters/_defCheckbox.scss:6]: @ebdabbs 

Thank you.

Comment (MaxwellKendall) [reply src/_scss/pages/search/filters/otherFilters/_defCheckbox.scss:6]: FWIW I added the border bottom for sake of uniformity of style; specifically when we show a selected pill like this

image


Here's NAICS:

image

**Comment by MaxwellKendall** [comment]
@ebdabbs 

01cce94 updates the tooltip copy [per this ticket](https://federal-spending-transparency.atlassian.net/browse/DEV-5738) that andrew just created.

**Pull request merged**